### DISCUSSION TITLE
微信群满了需要拉，求加好友拉我

### DISCUSSION THREAD

**Discussion by simonsheng1999** [chatchat-space/Langchain-Chatchat#457] [General]
之前跑过glm10b 130b chatglm6b
想看看能否把finetune加到里边去，langchain+finetune效果应该更好
微信号：ximingsheng1999

**Comment by radarfyh** [comment]
同求：微信号：hunter_coming

**Comment by Sersnow98** [comment]
同求，部署已完成，想和大家进行效果和best practice的探求
微信号：yjz244596848

**Reply by Sersnow98** [reply]
才看到并没有满。。已进

**Reply by leakers** [reply]
求拉群，julianbond

**Reply by PATRICK-1184** [reply]
求拉 zhaolei0071

**Comment by liangwei-artefact** [comment]
我也要加入 nightelfamber 想开发接入meta Llama 2 模型
